core/templates/components/summary-tile/subtopic-summary-tile.component.html

```diff
@@ -26,7 +28,7 @@
   }
 
   .subtopic-card:focus {
-    outline: 0;
+    outline: 2px solid black;
```

Comment:
We do not use color names. We always use rgb() or hex color code. See [coding style guidelines](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)

Reply:
Done